doc/bugs.md

```diff
@@ -55,6 +55,7 @@ See [list of issues](issues.md) for details.
 |           Summary                |            Credits         |         CVE       |       Upstream Acknowledgement          | Tests                                  |
 |:---------------------------:     |:--------------------------:|:-----------------:|:--------------------------------------: |:---------------------------------------:
 | GCM wrapped around counter       | Quan Nguyen                |  N/A              |   goo.gl/OdhZcY  |
+| P-384 and P-521 ScalarMult DoS   | Daniel Bleichenbacher      |  CVE-2019-6486    |  [golang/go#29903](https://github.com/golang/go/issues/29903) | To be released
```

Comment:
I think this bug was discovered by Harris Baskaran who ran Wycheproof against Golang.